executor/write.go

```diff
@@ -903,6 +918,15 @@ func (e *InsertValues) fillRowData(cols []*table.Column, vals []types.Datum, ign
 	if err = table.CastValues(e.ctx, row, cols, ignoreErr); err != nil {
 		return nil, errors.Trace(err)
 	}
+	for i, expr := range e.GenExprs {
+		var val types.Datum
+		val, err = expr.Eval(row)
+		if err != nil {
```

Comment:
Here need to consider the value of `ignoreErr`.
You'd better add some test about `load data`.